aspnetcore/mvc/views/view-compilation.md

```diff
@@ -1,66 +1,92 @@
 ---
-title: Razor view compilation and precompilation in ASP.NET Core
+title: Razor file compilation and precompilation in ASP.NET Core
 author: rick-anderson
-description: Learn how to enable MVC Razor view compilation and precompilation in ASP.NET Core apps.
+description: Learn about the benefits of precompiling Razor files and how to accomplish Razor file precompilation in an ASP.NET Core app.
 manager: wpickett
+monikerRange: '>= aspnetcore-1.1'
 ms.author: riande
-ms.date: 12/13/2017
+ms.custom: mvc
+ms.date: 05/17/2018
 ms.prod: asp.net-core
 ms.technology: aspnet
 ms.topic: article
 uid: mvc/views/view-compilation
 ---
-# Razor file (.cshtml) compilation in ASP.NET Core
+# Razor file compilation in ASP.NET Core
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-Razor views are compiled at runtime when the view is invoked. ASP.NET Core 2.1.0 or later compile views at build and publish time using [Razor Sdk](/aspnetcore/mvc/razor-pages/sdk). In ASP.NET Core 1.1, and ASP.NET Core 2.0, views can optionally be compiled at publish and deployed with the app &mdash; using the precompilation tool. 
+::: moniker range="= aspnetcore-1.1"
```

Comment:
Out of curiosity, do these appear like tabs? Should this in that case be ordered in descending (2.1 before 2.0)

Reply:
These don't appear as tabs. We're moving away from that approach, now that we have a version selector at the top of the docs page. Only content for the selected version is displayed.